packages/fxa-content-server/app/tests/spec/models/account.js

```diff
@@ -72,12 +72,12 @@ describe('models/account', function () {
     };
     ecosystemAnonIdPublicKeys = [
       {
-        'kty': 'EC',
+        kty: 'EC',
       },
       {
-        'kty': 'RSA',
+        kty: 'RSA',
```

Comment:
I made this key `RSA` so there'd be difference between the two objects in case there was anything worth testing about picking one of the two keys but eh, don't think we need to and it looks like we'll only ever use `EC` so we _could_ update it to more accurately mock what these will be but it also doesn't really matter. 🤷